### PR TITLE
docs(README): Add mention of Prisma Accelerate and Pulse

### PR DESCRIPTION
**Pull request by petradonka**
Short & concise. Let me know what you think.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +35,10 @@ Prisma is a **next-generation ORM** that consists of these tools:
 
 Prisma Client can be used in _any_ Node.js or TypeScript backend application (including serverless applications and microservices). This can be a [REST API](https://www.prisma.io/docs/concepts/overview/prisma-in-your-stack/rest), a [GraphQL API](https://www.prisma.io/docs/concepts/overview/prisma-in-your-stack/graphql), a gRPC API, or anything else that needs a database.
 
+**The Prisma ORM can also further be extended with these Prisma solutions:**
```
Comment (janpio) [inline README.md:38]: "Prisma solution"? Not "Prisma products"?
```suggestion
**The Prisma ORM can also further be extended with these Prisma products:**
```

File: README.md [hunk 2]
```diff
@@ -38,6 +35,10 @@ Prisma is a **next-generation ORM** that consists of these tools:
...
+- [Prisma Accelerate](https://prisma.io/docs/data-platform/accelerate/what-is-accelerate?utm_source=github&utm_medium=org-readme): Global database cache with scalable connection pooling
+- [Prisma Pulse](https://www.prisma.io/docs/data-platform/pulse/what-is-pulse?utm_source=github&utm_medium=org-readme): Real-time database events with type-safe subscriptions
```
Comment (janpio) [inline README.md:40]: Do these `utm` links allow us to track the number of clicks these get reliably?

Comment (janpio) [inline README.md:40]: I think `org-readme` is wrong here.

File: README.md (hunk 1 above)
Comment (petradonka) [reply README.md:38]: I had that first, more than happy with it :)

File: README.md (hunk 2 above)
Comment (petradonka) [reply README.md:40]: Yes, but this reminds me that I need to adjust them

Comment (petradonka) [reply README.md:40]: Yes, I missed that, sorry about that.

Comment (petradonka) [reply README.md]: Done!

Comment (meletj) [reply README.md]: Actually a pris.ly here could work better if you want to know the absolute number of clicks. And we could still assign UTM parameters.

**Review by janpio — APPROVED** [review]

**Pull request merged**